### ISSUE TITLE
Article does not show up in Safari

### ISSUE THREAD

**Issue by yuxin1234** [sstur/draft-js-utils#260]
Article does not show up in Safari, while it works fine in Chrome and Edge.
The error is "Invalid regular expression: invalid group specifier name"

Root cause: Safari doesn't support regex lookbehind. https://stackoverflow.com/questions/51568821/works-in-chrome-but-breaks-in-safari-invalid-regular-expression-invalid-group

**Comment by yuxin1234** [comment]
@sstur could you please add me as a contributor to the repo? I'll create a PR to fix the issue.

**Comment by sstur** [comment]
Hey @yuxin1234, thanks for offering to help. I believe you can make a PR without any special access, let me know if you run into any issues.

BTW, I think this issue was caused by #254, so feel free to make a PR to revert that, or if you have a solution that preserves the italic fix, even better. Thanks!

**Comment by sstur** [comment]
Note: this issue was previously being tracked in #255.

**Comment by yuxin1234** [comment]
@sstur Thanks. I have a fix that preserves the italic fix. I got this error when I did git push.
git push --set-upstream origin fixSafariDisplay
remote: Permission to sstur/draft-js-utils.git denied to yuxin1234.
fatal: unable to access 'https://github.com/sstur/draft-js-utils.git/': The requested URL returned error: 403
Please help. Thank you.

**Comment by sstur** [comment]
Right. When making a PR to a public/open-source repo you'll need to fork it first.

**Comment by sstur** [comment]
More details here: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork

**Comment by yuxin1234** [comment]
@sstur Thanks. My first time contributing to a public repo. Will follow the instructions.

**Comment by yuxin1234** [comment]
@sstur Please review https://github.com/sstur/draft-js-utils/pull/261. Thank you.
